docs/about-us.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: About Us
+permalink: /about/
+---
+
+*Beyond Methane RVA* is a community led campaign of volunteers that focuses on the Richmond Gas Works, our municipal gas 
+utility. Our work seeks to broaden public awareness of the health, safety, and economic impacts of Richmond Gas Works. 
+Beyond Methane RVA consists of a myraid of environmental groups and individuals alike. 
+
+Members of our organization donate their time and expertise to this campaign as volunteers. With our collective efforts 
+we hope to expedite cheap clean energy expansion and phase out of dirty natural gas. We believe that 
```

Comment:
I think we should use "affordable" instead of "cheap" wherever possible. We want to frame it positively whereas "cheap" has a negative connotation.

Reply:
Also, I just noticed that you have new lines as if this were code. This can be a hassle to maintain so I'd prefer to keep paragraphs on a single line and let the text editor wrap it for us.

Reply:
Updated.